extensions/sql-database-projects/src/dialogs/updateProjectFromDatabaseDialog.ts

```diff
@@ -365,17 +365,23 @@ export class UpdateProjectFromDatabaseDialog {
 	private createProjectLocationRow(view: azdata.ModelView): azdata.FlexContainer {
 		const browseFolderButton: azdata.Component = this.createBrowseFileButton(view);
 
-		const value = this.project ? this.project.projectFilePath : '';
+		let values: string[] = [];
+		this.workspaceProjects.forEach(projectUri => {
+			values.push(projectUri.fsPath as string);
```

Comment:
is the `as string` needed? The type of `fspath` should be string

Reply:
Yes, it is string. It wasn't necessary, removed it.